src/main/java/lesson5/task1/Text.java

```diff
@@ -0,0 +1,33 @@
+package lesson5.task1;
+
+import java.util.Scanner;
+
+public class Text {
+
+    private Scanner textScan = new Scanner(System.in);
+
+    public String checkPalindrome() {
+        System.out.println("Enter a text to be checked:");
+        String inputText = textScan.nextLine();
+        if (inputText.isEmpty()) {
+            return "Text is Empty!";
+        } else if (isPalindrome(inputText)) {
+            return "Text is a palindrome and the length of the text is " + inputText.length();
+        } else {
+            return "Text is not a palindrome!";
+        }
+    }
+
+    public boolean isPalindrome(String newText) {
```

Comment:
It would be good to add some unit tests here

Reply:
Unit test class created. Please review.